### DISCUSSION TITLE
Add Generate Master Key button to User page

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5591] [Feature Requests]
Currently, the only way to generate a master key is for the person who created the user to log out of specify, then log into 7 as the user, go to their username, click generate master key, then enter the user's password, then copy the master key.
It would be significantly easier and faster for IA's or collection managers to have a button on the user page itself so they can create it at the same time they create the user.  THen they can send all the information the user needs to log in at the same time.

**Comment by grantfitzsimmons** [comment]
Would be nice, but hopefully this will be needed less and less.
